projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -2541,13 +2572,6 @@ describe('igxCombo', () => {
                 expect(combo.valid).toEqual(IgxComboState.INITIAL);
                 expect(combo.comboInput.valid).toEqual(IgxInputState.INITIAL);
             });
-            it('should not open on click if combo is disabled', () => {
```

Comment:
Does this behavior change?

Reply:
I've just made and old test to be a unit test.